app/controllers/thesis_controller.rb

```diff
@@ -56,7 +56,11 @@ def publish_to_dspace
     if params[:graduation] == 'all' || params[:graduation].nil?
       flash[:warning] = 'Please select a term before attempting to publish theses to DSpace@MIT.'
     else
-      flash[:success] = 'If publishing to DSpace@MIT worked, this would be a meaningful update message.'
+      theses = filter_theses_by_term(Thesis.in_review).to_a
```

Comment:
This structure - looking up the list in the thesis controller, using the `in_review` scope - makes a lot of sense to me. Could I ask that the commented-out code above this block be removed as part of this PR?